src/internal/EnhancedButton.js

```diff
@@ -234,19 +232,21 @@ class EnhancedButton extends Component {
     }
   };
 
+  /*
   handleClick = (event) => {
```

Comment:
Yep, sorry forgot about that - was the first file I worked on. It was seemingly never being used before either since everything was handleTouchTap or similar. Couldn't see why it was there in the first place. I can remove it in a bit.